crates/router/src/core/payment_methods/cards.rs

```diff
@@ -282,10 +290,13 @@ pub async fn add_card_hs(
     card: api::CardDetail,
     customer_id: String,
     merchant_account: &domain::MerchantAccount,
+    locker_choice: &api_enums::LockerChoice,
+    card_reference: Option<&String>,
```

Comment:
this can also be `&str`
and the locker choice doesn't need to be `&`, as its a enum without any data it can implement copy and that won't affect performance